### DISCUSSION TITLE
ExpandedController in Expo plugin options?

### DISCUSSION THREAD

**Discussion by fredrikburmester** [react-native-google-cast/react-native-google-cast#537] [General]
Hi, 

I'm using expo and was wondering if there is a way to enable the ExpandedController without having to go in to the AppDelegate.mm file? Maybe an option in the plugin options?

Thanks.

**Comment by petrbela** [comment]
Hi, this will need to be implemented #538

**Reply by fredrikburmester** [reply]
Thank you for taking the time and creating an issue from this.
